webpack.common.config.js

```diff
@@ -127,12 +127,14 @@ var resolve = {
         'wikiPage': addonsPath('wiki/static/wikiPage.js'),
         'highlight-css': nodePath('highlight.js/styles/default.css'),
         'pikaday-css': nodePath('pikaday/css/pikaday.css'),
+        'keen-js': nodePath('keen-js'),
```

Comment:
This is indeed unnecessary. keen-js is installed with npm and properly declares the `browser` entry in its package.json (https://github.com/keen/keen-js/blob/master/package.json#L5).